docs/docsite/Makefile

```diff
@@ -97,16 +101,19 @@ core_structure: core_generate_rst
 gettext_structure: gettext_generate_rst
 	@echo "Creating symlinks in gettext_generate_rst"
 	-ln -sf ../rst/core_index.rst rst/index.rst
-	-ln -sf ../sphinx_conf/core_conf.py rst/conf.py
+	-ln -sf ../sphinx_conf/all_conf.py rst/conf.py
 
 gettext: gettext_structure
 	CPUS=$(CPUS) $(MAKE) -f Makefile.sphinx gettext
+	# if msgcat is installed handle all indexes, otherwise use the index from gettext_structure.
+	-msgcat "$(POTDIR)/core_index.pot" "$(POTDIR)/ansible_index.pot" "$(POTDIR)/2.10_index.pot" > "$(POTDIR)/tmp_index.pot" && mv "$(POTDIR)/tmp_index.pot" "$(POTDIR)/index.pot"
```

Comment:
Should we make this 2.10 the result of a dynamic command so we don't have to keep thinking about it in the future ?

Reply:
Spredzy and I talked about this on slack.  2.10 is a special case.  We have it for backwards compatibility only. (2.10_conf.py builds nearly all of the documentation as one website.  core_conf.py builds a subset for the /ansible-core/ site.  ansible_conf.py builds a subset for /ansible/3/ and beyond).  We're not going to have a 2.11_conf.py or a 3_conf.py; those versions are taken care of by the core_conf.py and ansible_conf.py.  So we shouldn't need to get the VERSION dynamically.